include/fs/fs.h

```diff
@@ -203,24 +204,28 @@ struct fs_file_system_t {
 /**
  * @brief File open
  *
- * Opens or creates, if does not exist, file depending on flags provided
- * and associates a stream with it.
+ * Opens or possibly creates a file and associates a stream with it.
  *
- * @param zfp Pointer to file object
- * @param file_name The name of file to open
+ * @details
+ * @p flags can be 0 or a binary combination of one or more of the following
+ * identifiers:
+ *   - <tt>FS_O_READ</tt>  open for read
```

Comment:
FWIW `@c FS_O_READ` would be the standard doxygen way to style code text.  `<tt>FS_O_READ | FS_O_WRITE</tt>` would be necessary for a multi-token sequence.

Reply:
Will fix.